.eslintrc.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
             {
                 "arrowParens": "always",
                 "bracketSpacing": true,
-                "jsxBracketSameLine": false,
+                "bracketSameLine": false,
```

Comment:
jsxBracketSameLine is [deprecated](https://prettier.io/blog/2021/09/09/2.4.0.html#replace-jsxbracketsameline-option-with-bracketsameline-option-11006httpsgithubcomprettierprettierpull11006-by-kurtztechhttpsgithubcomkurtztech)